src/main/java/mate/academy/spring/controller/AuthenticationController.java

```diff
@@ -23,7 +23,8 @@ public AuthenticationController(AuthenticationService authService,
 
     @PostMapping("/register")
     public UserResponseDto register(@RequestBody @Valid UserRequestDto requestDto) {
-        User user = authService.register(requestDto.getEmail(), requestDto.getPassword());
+        User user = authService.register(requestDto.getEmail(), requestDto.getPassword(),
+                requestDto.getRoles());
```

Comment:
DTO should not have a role.
[checklist](https://mate-academy.github.io/jv-program-common-mistakes/java-spring/security-part-2/jv-spring-security-checklist)
3. Add USER role to user at Authentication **service** in the method register().